From 176ad7fde4aec9b85fc5a15a4097e52035f4f6e6 Mon Sep 17 00:00:00 2001
From: fli <fupan.li@windriver.com>
Date: Wed, 14 Dec 2016 21:59:10 -0800
Subject: [PATCH] mwifiex: allow using network namespaces

This finally opens up the ability to put mwifiex
wiphy device into different network namespaces, just
as the other mac80211 devices did.

Signed-off-by: fli <fupan.li@windriver.com>
---
 drivers/net/wireless/marvell/mwifiex/cfg80211.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/wireless/marvell/mwifiex/cfg80211.c
+++ b/drivers/net/wireless/marvell/mwifiex/cfg80211.c
@@ -4266,6 +4266,7 @@ int mwifiex_register_cfg80211(struct mwi
 			WIPHY_FLAG_SUPPORTS_SCHED_SCAN |
 			WIPHY_FLAG_HAS_REMAIN_ON_CHANNEL |
 			WIPHY_FLAG_HAS_CHANNEL_SWITCH |
+			WIPHY_FLAG_NETNS_OK |
 			WIPHY_FLAG_PS_ON_BY_DEFAULT;
 
 	if (ISSUPP_TDLS_ENABLED(adapter->fw_cap_info))
